src/reporting/lambda/custom-resource/quicksight/index.ts

```diff
@@ -826,6 +886,12 @@ const updateDashboard = async (quickSight: QuickSight, commonParams: ResourceCom
     logger.info(`update dashboard finished. id: ${dashboardId}`);
 
     await waitForDashboardChangeCompleted(quickSight, commonParams.awsAccountId, dashboardId);
+
+    const versionNumber = dashboard.VersionArn?.substring(dashboard.VersionArn?.lastIndexOf('/') + 1);
```

Comment:
If the dashboard does not have `VersionArn` field, the `versionNumber` is `undefined`

Reply:
While the definition is an optional parameter, in reality, it should not be empty. If it is empty, an exception would have been thrown earlier.  If we want to fix this case, the only option is to throw an exception here。

There are no detail introduction about 'VersionArn' in document.
https://docs.aws.amazon.com/quicksight/latest/APIReference/API_UpdateDashboard.html